### PR TITLE
German (de) Translation

### PR DESCRIPTION
**Pull request by joschygit**
Added German Translation

### CODE REVIEW

File: Europe/de.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  "1_Month": "1 Monat",
+  "1_Year": "1 Jahr",
+  "2_Years": "2 Jahre",
+  "3_Months": "3 Monate",
+  "3_Years": "3 Jahre",
+  "5_Years": "5 Jahre",
+  "6_Months": "6 Monate",
+  "About": "Über",
+  "Add_Gas_Record": "Tankung hinzufügen",
+  "Add_New_Gas_Record": "Neue Tankung hinzufügen",
```
Comment (rbnhln) [inline Europe/de.json:24]:
```suggestion
  "Add_New_Gas_Record": "Neue Tankstand hinzufügen",
```

File: Europe/de.json [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Note": "Neue Notiz",
+  "Add_New_Odometer_Record": "Neuen Kilometerstand",
```
Comment (rbnhln) [inline Europe/de.json:26]:
```suggestion
  "Add_New_Odometer_Record": "Neuen Kilometerstand hinzufügen",
```

File: Europe/de.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Plan_Record": "Neuen Planungspunkt",
```
Comment (rbnhln) [inline Europe/de.json:27]:
```suggestion
  "Add_New_Plan_Record": "Neuen Planungspunkt hinzufügen",
```

File: Europe/de.json [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Reminder": "Neue Erinnerung",
+  "Add_New_Repair_Record": "Neue Reparatur",
```
Comment (rbnhln) [inline Europe/de.json:29]:
```suggestion
  "Add_New_Repair_Record": "Neue Reparatur hinzufügen",
```

File: Europe/de.json [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Service_Record": "Neue Wartung",
```
Comment (rbnhln) [inline Europe/de.json:30]:
```suggestion
  "Add_New_Service_Record": "Neuen Wartungseintrag hinzufügen",
```

File: Europe/de.json [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Supply_Record": "Neues Ersatzteil",
```
Comment (rbnhln) [inline Europe/de.json:31]:
```suggestion
  "Add_New_Supply_Record": "Neues Ersatzteil hinzufügen",
```

File: Europe/de.json [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+  "3_Years": "3 Jahre",
+  "5_Years": "5 Jahre",
+  "6_Months": "6 Monate",
+  "About": "Über",
+  "Add_Gas_Record": "Tankung hinzufügen",
+  "Add_New_Gas_Record": "Neue Tankung hinzufügen",
+  "Add_New_Note": "Neue Notiz",
+  "Add_New_Odometer_Record": "Neuen Kilometerstand",
+  "Add_New_Plan_Record": "Neuen Planungspunkt",
+  "Add_New_Reminder": "Neue Erinnerung",
```
Comment (rbnhln) [inline Europe/de.json:28]:
```suggestion
  "Add_New_Reminder": "Neue Erinnerung hinzufügen",
```

File: Europe/de.json [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Repair_Record": "Neue Reparatur",
+  "Add_New_Service_Record": "Neue Wartung",
+  "Add_New_Supply_Record": "Neues Ersatzteil",
+  "Add_New_Tax_Record": "Neuer KFZ-Steuereintrag",
```
Comment (rbnhln) [inline Europe/de.json:32]:
```suggestion
  "Add_New_Tax_Record": "Neuer Kfz-Steuereintrag hinzufügen",
```

File: Europe/de.json [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Upgrade_Record": "Neues Upgrade",
```
Comment (rbnhln) [inline Europe/de.json:33]:
```suggestion
  "Add_New_Upgrade_Record": "Neues Upgrade hinzufügen",
```

File: Europe/de.json [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_New_Vehicle": "Neues Fahrzeug",
```
Comment (rbnhln) [inline Europe/de.json:34]:
```suggestion
  "Add_New_Vehicle": "Neues Fahrzeug hinzufügen",
```

File: Europe/de.json [hunk 11]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_Note": "Notiz hinzufügen",
+  "Add_Odometer_Record": "Kilometerstand hinzufügen",
+  "Add_Plan_Record": "Planeintrag hinzufügen",
+  "Add_Reminder": "Erinnerung hinzufügen",
+  "Add_Repair_Record": "Reparatur hinzufügen",
+  "Add_Service_Record": "Wartung hinzufügen",
+  "Add_Supply_Record": "Ersatzteil hinzufügen",
+  "Add_Tax_Record": "KFz-Steuereintrag hinzufügen",
```
Comment (rbnhln) [inline Europe/de.json:42]:
```suggestion
  "Add_Tax_Record": "Kfz-Steuereintrag hinzufügen",
```

File: Europe/de.json [hunk 12]
```diff
@@ -0,0 +1,230 @@
...
+  "Add_Upgrade_Record": "Upgrade hinzufügen",
+  "Admin_Panel": "Admin-Panel",
+  "Amount_of_gas_refueled": "Menge Tankung",
```
Comment (rbnhln) [inline Europe/de.json:45]:
```suggestion
  "Amount_of_gas_refueled": "Menge Tankfüllung",
```

File: Europe/de.json [hunk 13]
```diff
@@ -0,0 +1,230 @@
...
+  "As_of_Today": "Stand Heute",
```
Comment (rbnhln) [inline Europe/de.json:46]:
```suggestion
  "As_of_Today": "Stand heute",
```

File: Europe/de.json [hunk 14]
```diff
@@ -0,0 +1,230 @@
...
+  "Consumption": "Verbrauch",
+  "Cost": "Kosten",
+  "Cost_of_gas_refueled": "Kosten der Tankfüllung",
+  "Cost_of_tax_paid": "Kosten der gezahlten Steuern",
+  "Cost_of_the_Plan": "Plankosten",
+  "Cost_of_the_repair": "Reparaturkosten",
+  "Cost_of_the_service": "Wartungskosten",
+  "Cost_of_the_upgrade/mods": "Kosten des Upgrades/der Modifikation",
+  "Critical": "Kritisch",
+  "Current_Stage": "Aktuelles Stadium",
```
Comment (rbnhln) [inline Europe/de.json:64]:
```suggestion
  "Current_Stage": "Aktuelles Stand",
```

File: Europe/de.json [hunk 15]
```diff
@@ -0,0 +1,230 @@
...
+  "Dark_Mode": "Dunkler Modus",
+  "Dashboard": "Dashboard",
+  "Date": "Datum",
+  "Date_purchased": "Kaufdatum",
+  "Date_recorded": "Aufnahmedatum",
+  "Date_refueled": "Tankdatum",
+  "Date_Refueled": "Datum der Betankung",
```
Comment (rbnhln) [inline Europe/de.json:71]:
```suggestion
  "Date_Refueled": "Tankdatum",
```

File: Europe/de.json [hunk 16]
```diff
@@ -0,0 +1,230 @@
...
+  "Date_service_was_performed": "Servicedatum",
+  "Date_tax_was_paid": "Datum der Steuerzahlung",
+  "Date_upgrade/mods_was_installed": "Datum der Installation des Upgrades/der Modifikation",
+  "Default_Tab": "Standard-Tab",
+  "Delete": "Löschen",
+  "Delete_Vehicle": "Fahrzeug löschen",
+  "Describe_the_Plan": "Beschreiben Sie die Planung",
+  "Description": "Beschreibung",
+  "Description_of_item(s)_repaired(i.e._Alternator)": "Beschreibung der reparierten Gegenstände (z.B. Lichtmaschine)",
+  "Description_of_item(s)_serviced(i.e._Oil_Change)": "Beschreibung der gewarteten Teile (z. B. Ölwechsel)",
```
Comment (rbnhln) [inline Europe/de.json:82]:
```suggestion
  "Description_of_item(s)_serviced(i.e._Oil_Change)": "Beschreibung der Wartung (z. B. Ölwechsel)",
```

File: Europe/de.json [hunk 17]
```diff
@@ -0,0 +1,230 @@
...
+  "Edit_Odometer_Record": "Kilomenterzählereintrag bearbeiten",
+  "Edit_Reminder": "Erinnerung bearbeiten",
+  "Edit_Repair_Record": "Reparatureintrag bearbeiten",
+  "Edit_Service_Record": "Wartungseintrag bearbeiten",
+  "Edit_Supply_Record": "Ersatzteileintrag bearbeiten",
+  "Edit_Tax_Record": "Steuereintrag bearbeiten",
+  "Edit_Upgrade_Record": "Upgrade bearbeiten",
+  "Edit_Vehicle": "Fahrzeug bearbeiten",
+  "Electric_Vehicle": "Elektrofahrzeug",
+  "Email": "e-Mail-Adresse",
```
Comment (rbnhln) [inline Europe/de.json:101]:
```suggestion
  "Email": "Email",
```

File: Europe/de.json [hunk 18]
```diff
@@ -0,0 +1,230 @@
...
+  "Enable_Authentication": "Authentifikation aktivieren",
+  "Enable_CSV_Imports": "CSV-Import aktivieren",
+  "Expenses_by_Month": "Ausgaben nach Monat",
+  "Expenses_by_Type": "Ausgabe nach Typ",
+  "Export_Attachments": "Anhänge exportieren",
+  "Export_to_CSV": "Nach CSV exportiern",
+  "Failure_to_format_the_data_correctly_can_cause_data_corruption._Please_make_sure_you_make_a_copy_of_the_local_database_before_proceeding.": "Wenn Sie die Daten nicht korrekt formatieren, kann dies zu fehlerhaften Daten führen. Bitte stellen Sie sicher, dass Sie eine Kopie der lokalen Datenbank erstellen, bevor Sie fortfahren",
```
Comment (rbnhln) [inline Europe/de.json:108]:
```suggestion
  "Failure_to_format_the_data_correctly_can_cause_data_corruption._Please_make_sure_you_make_a_copy_of_the_local_database_before_proceeding.": "Sollten die Daten nicht korrekt formatiert sein, kann dies zu fehlerhaften Daten führen. Bitte stellen Sie sicher, dass Sie eine Kopie der lokalen Datenbank erstellt haben, bevor Sie fortfahren.",
```

File: Europe/de.json [hunk 19]
```diff
@@ -0,0 +1,230 @@
...
+  "Fuel_Mileage_by_Month": "Kraftstoff-Kilometerstand nach Monat",
+  "Future_Date": "Zukünftiges Datum",
+  "Future_Odometer_Reading": "Zukünftiger Kilometerstand",
+  "Garage": "Garage",
+  "Gasoline": "Benzin",
+  "Generate_User_Token": "Benutzer-Token generieren",
+  "Import": "Importieren",
+  "Import_Data_from_CSV": "Daten aus CSV importieren",
+  "Import_via_CSV": "Importieren über CSV",
+  "In_order_for_this_utility_to_function_properly,_your_CSV_file_MUST_be_formatted_exactly_like_the_provided_sample._Dates_must_be_supplied_in_a_string._Numbers_must_be_supplied_as_numbers_without_currency_formatting.": "Damit dieses Dienstprogramm richtig funktioniert, MUSS Ihre CSV-Datei genau so formatiert sein wie das mitgelieferte Beispiel.",
```
Comment (rbnhln) [inline Europe/de.json:121]:
```suggestion
  "In_order_for_this_utility_to_function_properly,_your_CSV_file_MUST_be_formatted_exactly_like_the_provided_sample._Dates_must_be_supplied_in_a_string._Numbers_must_be_supplied_as_numbers_without_currency_formatting.": "Damit dieser Vorgang richtig funktioniert, MUSS Ihre CSV-Datei genau so formatiert sein, wie das angegebene Beispiel. Daten müssen als string und Geldbeträge ohne Währungssymbol angegeben werden.",
```

File: Europe/de.json [hunk 20]
```diff
@@ -0,0 +1,230 @@
...
+  "In_Stock": "Auf Lager",
+  "Input_Gas_Consumption_in_Liters,_it_will_be_converted_to_UK_Gals_for_MPG_Calculation": "Kraftstoffverbrauch in Litern, wird für die MPG-Berechnung in UK-Gallonen umgerechnet",
+  "Is_Admin": "Ist Administartor",
+  "Is_Filled_To_Full": "Vollbetankung",
+  "Is_Recurring": "Wiederkehrend",
+  "Issued_To": "Ausgestellt für",
+  "Language": "Sprache",
+  "Last_Reported_Odometer_Reading": "Letzter Tachostand",
```
Comment (rbnhln) [inline Europe/de.json:129]:
```suggestion
  "Last_Reported_Odometer_Reading": "Letzter Kilometerstand",
```

File: Europe/de.json [hunk 21]
```diff
@@ -0,0 +1,230 @@
...
+  "Manage_Vehicle": "Fahrzeuge verwalten",
+  "Max_File_Size:_28.6MB": "Maximale Dateigröße: 28,6 MB",
+  "Max_Fuel_Economy": "Maximaler Kraftstoffverbrauch",
+  "Metric": "Metrik",
+  "Min_Fuel_Economy": "Minimaler Kraftstoffverbrauch",
+  "Missed_Fuel_Up(Skip_MPG_Calculation)": "Verpasste Betankung (Überspringen der MPG-Berechnung)",
+  "Model": "Model",
+  "Month": "Monat",
+  "Move_To": "Verschieben nach",
+  "No_data_found,_create_reminders_to_see_visualizations_here.": "Keine Daten, erstellen Sie Erinnerungen, um Visualisierungen zu sehen.",
```
Comment (rbnhln) [inline Europe/de.json:144]:
```suggestion
  "No_data_found,_create_reminders_to_see_visualizations_here.": "Keine Daten gefunden, erstellen Sie Erinnerungen, um eine Visualisierungen zu sehen.",
```

File: Europe/de.json [hunk 22]
```diff
@@ -0,0 +1,230 @@
...
+  "No_data_found,_insert/select_some_data_to_see_visualizations_here.": "NKeine Daten, geben Sie einige Daten ein/wählen Sie einige Daten aus, um Visualisierungen anzuzeigen.",
```
Comment (rbnhln) [inline Europe/de.json:145]:
```suggestion
  "No_data_found,_insert/select_some_data_to_see_visualizations_here.": "Keine Daten gefunden, geben Sie einige Daten ein/wählen Sie einige Daten aus, um Visualisierungen anzuzeigen..",
```

File: Europe/de.json [hunk 23]
```diff
@@ -0,0 +1,230 @@
...
+  "Odometer_reading": "Kilometerstand",
+  "Odometer_Reading": "Kilometerstand",
+  "Odometer_reading_when_refueled": "Kilometerstand bei Betankung",
+  "Odometer_reading_when_repaired": "Kilometerstand bei Reparatur",
+  "Odometer_reading_when_serviced": "Kilometerstand bei Service",
+  "Odometer_reading_when_upgraded/modded": "Kilometerstand bei Upgrade/Modifikation",
+  "Only_when_Adding_Service/Repair/Upgrade/Fuel_Record_or_Completing_a_Plan": "Nur bei Hinzufügen von Service/Reparatur/Upgrade/Tankprotokoll oder Abschluss eines Plans",
+  "Other": "Sonstiges",
+  "Part_#": "Teilenummer",
+  "Part_#/Model_#/SKU_#": "Teilenummer/Modellnummer/SKU-Nummer",
```
Comment (rbnhln) [inline Europe/de.json:162]:
```suggestion
  "Part_#/Model_#/SKU_#": "Teile-/Modell-/SKU-Nummer",
```

File: Europe/de.json [hunk 24]
```diff
@@ -0,0 +1,230 @@
...
+  "Quantity": "Menge",
+  "Remind_me_on": "Erinnere mich am",
+  "Reminder": "Erinnerung",
+  "Reminder_Description": "Beschreibung der Erinnerung",
+  "Reminders": "Erinnerungen",
+  "Reminders_by_Category": "Erinnerungen nach Kategorie",
+  "Reminders_by_Urgency": "Erinnerungen nach Dringlichkeit",
+  "Repair": "Reparatur",
+  "Repairs": "Reparaturen",
+  "Replace_$0.00_Costs_with_---": "$0,00 Kosten durch --- ersetzen",
```
Comment (rbnhln) [inline Europe/de.json:180]:
```suggestion
  "Replace_$0.00_Costs_with_---": "0,00€ Kosten durch --- ersetzen",
```

File: Europe/de.json [hunk 25]
```diff
@@ -0,0 +1,230 @@
...
+  "Select_Supplies": "Lieferungen auswählen",
+  "Service": "Service",
+  "Service_Record": "Serviceaufzeichnung",
+  "Service_Records": "Serviceaufzeichnungen",
+  "Settings": "Einstellungen",
+  "Sort_lists_in_Descending_Order(Newest_to_Oldest)": "Listen in absteigender Reihenfolge sortieren (neueste zu älteste)",
+  "Supplier": "Lieferant",
+  "Supplier/Vendor": "Lieferant/Verkäufer",
+  "Supplies": "Ersatzteile",
+  "Supplies_are_requisitioned_immediately_after_the_record_is_created_and_cannot_be_modified._If_you_have_incorrectly_entered_the_amount_you_needed_you_will_need_to_correct_it_in_the_Supplies_tab.": "Ersatzteile werden sofort nach Erstellung des Eintrags angefordert und können nicht geändert werden. Wenn Sie die benötigte Menge falsch eingegeben haben, müssen Sie dies im Register 'Ersatzteile' korrigieren.",
```
Comment (rbnhln) [inline Europe/de.json:194]:
```suggestion
  "Supplies_are_requisitioned_immediately_after_the_record_is_created_and_cannot_be_modified._If_you_have_incorrectly_entered_the_amount_you_needed_you_will_need_to_correct_it_in_the_Supplies_tab.": "Ersatzteile werden sofort nach Erstellung des Eintrags eingetragen und können nicht geändert werden. Wenn Sie die benötigte Menge falsch eingegeben haben, müssen Sie dies im Register 'Ersatzteile' korrigieren.",
```

**Review by rbnhln — COMMENTED** [review]
Made a few corrections and suggestions.

**Comment by hargata** [comment]
is this ready to be merged?

**Comment by joschygit** [comment]
> is this ready to be merged?

Yes, i think its ok so far. Thanks a lot.

**Pull request merged**